beacon-chain/state/state-native/getters_validator.go

```diff
@@ -223,6 +224,36 @@ func (b *BeaconState) PubkeyAtIndex(idx primitives.ValidatorIndex) [fieldparams.
 	return bytesutil.ToBytes48(v.PublicKey)
 }
 
+// AggregatedKeyFromIndices builds an aggregated public key from the provided
+// validator indices.
+func (b *BeaconState) AggregatedKeyFromIndices(idxs []uint64) (bls.PublicKey, error) {
+	b.lock.RLock()
+	defer b.lock.RUnlock()
+
+	pubKeys := make([][]byte, len(idxs))
+	for i, idx := range idxs {
+		var v *ethpb.Validator
+		if features.Get().EnableExperimentalState {
```

Comment:
This case is not tested at all, is it expected?

Reply:
I think its fine in this particular case as its simply for the experimental state, we don't do the same for a few other methods too. Also the experimental state has been tested in e2e, so these paths are tested, just not in the current unit tests. Once we flip the flag, it will get tested as it becomes the default path